test/quantization/test_quant_api.py

```diff
@@ -197,13 +197,13 @@ def test_gptq_quantizer(self):
         with torch.no_grad():
             compiled(inputs[0].values[0], inputs[1].values[0])
 
-    @unittest.skip("skipping until we get checkpoints for gpt-fast")
+    # @unittest.skip("skipping until we get checkpoints for gpt-fast")
     def test_gptq_quantizer_gpt_fast(self):
         from torchao.quantization.GPTQ import Int8DynActInt4WeightGPTQQuantizer, InputRecorder
         # should be similar to TorchCompileDynamicQuantizer
         precision = torch.bfloat16
         device = "cuda"
-        checkpoint_path = Path("../gpt-fast/checkpoints/meta-llama/Llama-2-7b-chat-hf/model.pth")
+        checkpoint_path = Path("/home/cdhernandez/local/gpt-fast/checkpoints/meta-llama/Llama-2-7b-chat-hf/model.pth")
```

Comment:
We definitely can't store this on CI. Maybe `open_llama_7b` works better. gpt-fast supports that as well.